templates/master/01-master-kubelet/_base/units/kubelet.service.yaml

```diff
@@ -38,6 +38,7 @@ contents: |
         --cloud-provider={{cloudProvider .}} \
         --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec \
         {{cloudConfigFlag . }} \
+        --hostname-override=${KUBELET_NODE_NAME} \
```

Comment:
`${KUBELET_NODE_NAME}` is possibly undefined, is it fine?

From the `e2e-metal-ipi` job I can see kubelet was started with `FLAG: --hostname-override=""` so I guess it's OK.

Reply:
Yes, this is fine. Also passing an empty override works the same as passing no override, and is documented to do this.